scripts/stac/imagery/tests/collection_test.py

```diff
@@ -177,6 +179,65 @@ def test_default_provider_is_present(metadata: CollectionMetadata) -> None:
     assert {"name": "Maxar", "roles": ["producer"]} in collection.stac["providers"]
 
 
+def test_capture_area_added(metadata: CollectionMetadata) -> None:
+    collection = ImageryCollection(metadata)
+    target = mkdtemp()
+
+    polygons = []
+    polygons.append(
+        shapely.geometry.shape(
+            {
+                "type": "MultiPolygon",
+                "coordinates": [
+                    [
+                        [
+                            [178.259659571653, -38.40831927359251],
+                            [178.26012930415902, -38.41478071250544],
+                            [178.26560430668172, -38.41453416326152],
+                            [178.26513409076952, -38.40807278109057],
+                            [178.259659571653, -38.40831927359251],
+                        ]
+                    ]
+                ],
+            }
+        )
+    )
+    polygons.append(
+        shapely.geometry.shape(
+            {
+                "type": "MultiPolygon",
+                "coordinates": [
+                    [
+                        [
+                            [178.25418498567294, -38.40856551170436],
+                            [178.25465423474975, -38.41502700730107],
+                            [178.26012930415902, -38.41478071250544],
+                            [178.259659571653, -38.40831927359251],
+                            [178.25418498567294, -38.40856551170436],
+                        ]
+                    ]
+                ],
+            }
+        )
+    )
+
+    collection.add_capture_area(polygons, target)
+
+    assert collection.stac["assets"]["capture_area"]["href"] == "./capture-area.geojson"
+    assert collection.stac["assets"]["capture_area"]["title"] == "Capture area"
+    assert collection.stac["assets"]["capture_area"]["type"] == "application/geo+json"
+    assert collection.stac["assets"]["capture_area"]["roles"] == ["metadata"]
+    assert StacExtensions.file.value in collection.stac["stac_extensions"]
+    assert "file:checksum" in collection.stac["assets"]["capture_area"]
+    assert (
+        collection.stac["assets"]["capture_area"]["file:checksum"]
+        == "12209043a7059c39aa8a8bd22ad0e50476b1628a5b20df11c5a507900cab19f2741b"
+    )
+    assert "file:size" in collection.stac["assets"]["capture_area"]
+    assert collection.stac["assets"]["capture_area"]["file:size"] == 341
+    rmtree(target)
```

Comment:
We should probably use `with mkdtemp() as target` to remove the directory if the test fails. Or do you want to keep it?